src/themes/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { StyleSheet } from 'react-native'
+
+const StyleUtils = StyleSheet.create( {
+  /**
+   * Set Top and Bottom margin to auto
+   */
+  mx: {
```

Comment:
What does `mx` mean? I had to visit this file to find out. I'm also a little sceptical about whether the two lines that this class has here is better than just inlining them in the actual style definitions themselves.

Thoughts @bhajneet?

Reply:
This feels more like something you would use on multiple items I suppose. Hence why it's probably in utils.ts, if it gets repeated around to other places then I don't think it makes sense to inline them? I have no idea how auto margins are being used in the front end

Reply:
`mx` is margin on x-axis. Something I found in here https://styled-system.com/table/ . Let me know if I should rename this?